docs/reference_guide.md

```diff
@@ -1829,6 +1831,42 @@ NetworkManager:1155 /var/lib/sss/mc/passwd (deleted)
 
 ```
 
+## 18. `rawtracepoint`: Static Tracing, Kernel-Level
+
+Syntax: `rawtracepoint:name` or `rt:name`
+
+These use tracepoints (a Linux kernel capability).
```

Comment:
This should mention the differences from `tracepoint`. Also, since these two are quite related, it'd make sense to put them next to each other in the reference guide.